specification/paths/Brokers-broker_id-relationships-carriers.json

```diff
@@ -0,0 +1,67 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/components/parameters/path-broker_id"
+    }
+  ],
+  "post": {
+    "tags": [
+      "Brokers"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "enterprise.manage"
+        ]
+      }
+    ],
+    "summary": "Add carrier relationships to a broker.",
+    "description": "This endpoint adds carrier relationships to the broker with the supplied id.",
+    "requestBody": {
+      "description": "The carrier relationships to be added to the broker.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "$ref": "#/components/schemas/CarrierRelationship"
```

Comment:
We should support adding multiple carriers at once, so let's make a `CarriersRelationship` component schema (for it to be recognized, the `CarriersRelationship` should also be mentioned in `specification/schemas.json`)